com.github.dahenson.agenda.json

```diff
@@ -19,56 +20,7 @@
 		"cflags": "-O2",
 		"cxxflags": "-O2"
 	},
-	"cleanup": [
```

Comment:
You still need some of the cleanup

Reply:
Hm I tried adding
```
	"cleanup": [
		"/bin/vala*",
		"/bin/vapi*",
		"/include",
		"/lib/pkgconfig",
		"/share/vala",
		"/share/vala-0.36",
		"*.a",
		"*.la"
	],
```
but nothing was deleted during the cleanup stage. Will check if that happens with other manifests as well.